test/Bugs/randombug.baseline

```diff
@@ -63,10 +63,10 @@ Test case 30
 Passed
 Test case 31
 Error
-	at func2 (randombug.js:287:9)

```

Comment:
Interesting. Three of those test folders also had their own version TrimStackTracePath which I refactored to the one file now found under UnitTestFramework in core.